optaweb-employee-rostering-backend/src/main/resources/solverConfig.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<solver>
+  <scoreDirectorFactory>
+    <scoreDrl>org/optaweb/employeerostering/service/solver/employeeRosteringScoreRules.drl</scoreDrl>
```

Comment:
@ge0ffrey Can we make this configurable with `application.properties`? Solver config XML could then be completely removed unless needed by power users.

Reply:
yes, good idea! Duncan asked something similar. Added to my todo list

Reply:
For now, quarkus and sprint-boot will support "constraints.drl" in the root package (PR being merged soon), but not support a application.properties yet because it's unclear whether that property key should contain "scoreDirectorFactory" or not. That's for phase 2.